functests/sctp/sctp_sriov.go

```diff
@@ -0,0 +1,202 @@
+package sctp
+
+import (
+	"context"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gstruct"
+
+	netattdefv1 "github.com/openshift/sriov-network-operator/pkg/apis/k8s/v1"
+	sriovv1 "github.com/openshift/sriov-network-operator/pkg/apis/sriovnetwork/v1"
+	sriovtestclient "github.com/openshift/sriov-network-operator/test/util/client"
+	sriovcluster "github.com/openshift/sriov-network-operator/test/util/cluster"
+	sriovdiscovery "github.com/openshift/sriov-network-operator/test/util/discovery"
+	sriovnamespaces "github.com/openshift/sriov-network-operator/test/util/namespaces"
+	sriovnetwork "github.com/openshift/sriov-network-operator/test/util/network"
+	runtimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/client"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/discovery"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/execute"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/namespaces"
+	"github.com/openshift-kni/cnf-features-deploy/functests/utils/sriov"
+)
+
+const (
+	sriovOperatorNamespace = "openshift-sriov-network-operator"
```

Comment:
lets allow to configure this variable just the same as we can in the sriov repo

Reply:
The dpdk one here is following this same approach, I would leave it as it is.
If we ever need to have it here too (read, if we want to test this on kind) we'll introduce it.